warehouse/templates/index.html

```diff
@@ -85,34 +90,30 @@ <h1 class="homepage-banner__title">Find, install and publish Python packages wit
 </section>
 
 <section class="horizontal-section horizontal-section--grey">
-  <div class="site-container">
-    <div class="col-half">
+  <div class="site-container" >
+    <div class="col-half" data-controller="show-trending-projects">
       <div class="heading-wsubtitle">
         <h2 class="heading-wsubtitle__heading">Trending Projects</h2>
         <p class="heading-wsubtitle__subtitle">Trending projects as downloaded by the community</p>
       </div>
       <div class="package-list">
         {% for release in trending_projects %}
-          {{ project_snippet(release) }}
+          {{ project_snippet(release, "trending") }}
         {% endfor %}
       </div>
-      {#
-      <a class="button" href="TODO">Show More</a>
-      #}
+      <button class="button" type="button" data-action="click->show-trending-projects#addFive" data-target="show-trending-projects.addFiveButton">Show More</button>
     </div>
-    <div class="col-half">
+    <div class="col-half" data-controller="show-latest-releases">
       <div class="heading-wsubtitle">
         <h2 class="heading-wsubtitle__heading">New Releases</h2>
         <p class="heading-wsubtitle__subtitle">Hot off the press: the newest project releases</p>
       </div>
       <div class="package-list">
         {% for release in latest_releases %}
-          {{ project_snippet(release) }}
+          {{ project_snippet(release, "latest") }}
```

Comment:
I think you can even remove the class parameter?

Reply:
not sure if that is possible, as I mentioned above